docs/manual/manual.adoc

```diff
@@ -1516,6 +1516,32 @@ the presence of a matching SWID tag:
 $ oscap oval eval --results results.xml r2860-rhel-oval.xml
 ----
 
+
+=== Notes on specifics of OVAL implementation
+
+==== unix-def:FileBehaviors with recurse_file_system="local"
```

Comment:
It's not just unix-def namespace, it's also in texfilecontent54 which belongs to independent and also others.

Maybe it would be better to start the paragraph with a sentence mentioning all OVAL tests that are affected. Them you can change the title to something like 

```suggestion
==== Excluding non-local filesystems using the recurse_file_system="local" attribute
```

Reply:
It is still not clear for me how the difference in `ind-def:FileBehaviors`, `win-def:FileBehaviors` and `unix-def:FileBehaviors` should be handled. But you're right, on linux we behave in the same way both for ind and unix.